packages/web/config/generate-chain-infos/source-chain-infos.ts

```diff
@@ -2613,6 +2613,15 @@ const mainnetChainInfos: SimplifiedChainInfo[] = [
         //coinGeckoId: "pool:usdx",
         coinImageUrl: "/tokens/usdx.svg",
       },
+      {
+        coinDenom: "USDT",
+        coinMinimalDenom: "erc20/tether/usdt",
+        coinDecimals: 6,
+        //coinGeckoId: "tether",
```

Comment:
@JeremyParish69 why do some of our assets have `//coinGeckoId` commented out? I notice some of them have different coinGeckoIds?

Reply:
CoinGeckoId is used for assigning a price to the asset. We used to use mostly ids that were actually from CoinGecko, but now we define custom ids (any id starting with 'pool:' is a custom id) in the price.ts file that looks at on-chain pool balances to derive a price.
If needed, it would be easier to switch to actual CoinGeckoIds if they are just commented out.
Also, for when we can separate chain suggestion from our displayed price, we'll want to use actual CoinGecko Ids (e.g., 'tether') for the Chain Suggestion to wallets, and we'll want to use our custom pricing for our frontend.